src/syscall.h

```diff
@@ -8,14 +8,36 @@ See the file LICENSE for details.
 #define SYSCALL_H
 
 #include "kerneltypes.h"
+#include "graphics.h"
 
 #define SYSCALL_exit     1
 #define SYSCALL_testcall 2
 #define SYSCALL_yield    3
 
+#define SYSCALL_window_create 200
+#define SYSCALL_window_set_border_color 201
+#define SYSCALL_window_draw_line 202
+#define SYSCALL_window_draw_arc 203
+#define SYSCALL_window_draw_circle 204
+#define SYSCALL_window_draw_char 205
+#define SYSCALL_window_draw_string 206
```

Comment:
Yeah but at the same time if we really adhere to a minimalistic philosophy when we design syscalls, we should only keep the ones we need.